### PR TITLE
Styles recovered. 

### PR DESCRIPTION
**Pull request by afhuertass**
Fixed mishap with the styles

### CODE REVIEW

File: assets/config.css [hunk 1]
```diff
@@ -40,7 +40,7 @@
     background-color: #f9f9f9;
 }
 
-#color-column, #size-column, #x-axis, #y-axis {
+#color-column, #size-column, #x-axis, #y-axis, #template-id  {
     width: 288px
 }
```
Comment (llmlks) [inline assets/config.css]: Sorry, didn't notice before, but this is still missing the last definition for 
```

.hidden {
    display: none;
}
```

**Review by llmlks — CHANGES_REQUESTED** [review]

File: assets/config.css (hunk 1 above)
Comment (afhuertass) [reply assets/config.css]: Added!

**Review by llmlks — CHANGES_REQUESTED** [review]
Still missing the `#opacity-unit` that should be after `#timestep-unit` on row 16

**Review by llmlks — APPROVED** [review]
Looks good, ready to merge

**Pull request merged**